### PR TITLE
fix default repos; add repos for lorax;

### PR DESCRIPTION
**Pull request by Kazero0**

### CODE REVIEW

File: tools/lorax/lorax.sh [hunk 1]
```diff
@@ -17,7 +17,11 @@ lorax --nomacboot --add-template=${ImageDir}/../tools/lorax/lorax.tmpl \
 --add-template-var=kern_cmdline="${Cmdline}" \
 --logfile=${ImageDir}/../lorax.log \
 -p "EulerOS V2.0SP3" -v iSula -r 1 --isfinal --buildarch=x86_64 \
--s ${RepoAddr} \
+-s $RepoAddr \
+-s http://mirror.centos.org/centos/7/updates/x86_64/ \
```
Comment (initlove) [inline tools/lorax/lorax.sh]: Why do we need to add centos repo?

File: config/atlas.yml [hunk 2]
```diff
@@ -57,7 +56,7 @@ rmpackages:
   - binutils
   - libdb-utils
 files:
-    add:
+    add: [["/tmp/euleros-antos-host.conf","/etc/ostree/remotes.d/euleros-antos-host.conf"]]
```
Comment (initlove) [inline config/atlas.yml]: add a /tmp/ file?

File: tools/lorax/lorax.sh (hunk 1 above)
Comment (Kazero0) [reply tools/lorax/lorax.sh]: It is hard to collect all packages for lorax building. When lorax building, it installs many tools into host machine which is not relevant to our OS. Collecting these packages and their deps is exhausted and meaningless to OS itself.

**Comment by initlove** [comment]
LGTM

**Pull request merged**